### PR TITLE
Add installation docs

### PR DESCRIPTION
**Pull request by guineveresaenger**
Part of pulumi/home#3598

Generates a single index.md file with relevant content.

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+The kafka provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/kafka`](https://www.npmjs.com/package/@pulumi/kafka)
+* Python: [`pulumi-kafka`](https://pypi.org/project/pulumi-kafka/)
+* Go: [`github.com/pulumi/pulumi-kafka/sdk/v3/go/kafka`](https://github.com/pulumi/pulumi-kafka)
+* .NET: [`Pulumi.Kafka`](https://www.nuget.org/packages/Pulumi.Kafka)
+* Java: [`com.pulumi/kafka`](https://central.sonatype.com/artifact/com.pulumi/kafka)
+## Overview
+<!-- schema generated by tfplugindocs -->
```
Comment (iwahbe) [inline docs/_index.md:16]: We shouldn't insert empty `## Overview` sections.

Comment (guineveresaenger) [reply docs/_index.md]: Arguably, this is an issue with the upstream docs being wrong/inclomplete - expecting an introduction seems reasonable.

Anyway. Addressed this in https://github.com/pulumi/pulumi-terraform-bridge/pull/2419/commits/415a9ca9f3c1bfa3e673aa81784f274ae0c20fb3 and applied here.

**Review by iwahbe — APPROVED** [review]
Was removing `## Overview` a manual change?

**Comment by guineveresaenger** [comment]
No, this was generated with https://github.com/pulumi/pulumi-terraform-bridge/commit/415a9ca9f3c1bfa3e673aa81784f274ae0c20fb3.

**Pull request merged**